packages/shared/src/features/onboarding/shared/StepHeadline.tsx

```diff
@@ -0,0 +1,50 @@
+import type { ReactElement } from 'react';
+import React from 'react';
+import classNames from 'classnames';
+import {
+  Typography,
+  TypographyType,
+  TypographyTag,
+} from '../../../components/typography/Typography';
+import type { TypographyProps } from '../../../components/typography/Typography';
+
+export enum StepHeadlineAlign {
+  Left = 'left',
+  Center = 'center',
+  Right = 'right',
+}
+
+export type StepHeadlineProps = {
+  align?: StepHeadlineAlign;
+  description?: string;
+  descriptionProps?: TypographyProps<TypographyTag.P>;
+  heading: string;
+};
+
+const StepHeadline = ({
+  align = StepHeadlineAlign.Center,
+  description,
+  descriptionProps,
+  heading,
+}: StepHeadlineProps): ReactElement => {
+  return (
+    <div
+      data-testid="step-headline-container"
+      className={classNames('flex flex-col gap-2', `text-${align}`)}
```

Comment:
```suggestion
      className={classNames('flex flex-col gap-2', `text-${align}`)}
```

This won't work in tailwind. You can't use dynamic classes like this.
You'll need to pass full `text-center` basically.

https://tailwindcss.com/docs/detecting-classes-in-source-files#dynamic-class-names